docs/safety.md

```diff
@@ -0,0 +1,113 @@
+# Taproot Assets Operational Safety Guidelines
+
+As of version `v0.3.0-alpha`, Taproot Assets can be used on Bitcoin's `mainnet`
+network. That means in any version after `v0.3.0` there won't be any breaking
+changes and any assets minted with that version should be future-proof.
+But signaling readiness for `mainnet` does NOT mean that there won't be any bugs
+or that all planned safety and backup measures are fully in place yet.
+
+**That means, special care must be taken to avoid loss of funds (both assets
+and BTC)!**
+
+## How to avoid loss of funds (short version, tl;dr)
+
+In short, there is no recovery mechanism in place yet that allows you to recover
+assets from only the `lnd` seed. So if you lose your `tapd`'s database, or it
+gets corrupted, you lose access to all assets minted or received by that `tapd`.
+In addition, you also cannot spend the BTC used to carry/anchor the assets.
+
+**To avoid loss of funds, make sure you back up your `/home/<user>/.tapd` folder
+regularly.** If you are using a Postgres database as the database backend, it
+is enough to make backups of that database.
```

Comment:
```suggestion
**To avoid loss of funds:
1) make sure the `/home/<user>/.tapd` directory is backed up regularly.** If `tapd` is configured to use Postgres as the database backend, backups this database is 
sufficient to preserve access to funds.
2)`lnd`'s seed phrase has been securely backed up, as all `tapd` assets private 
keys are derived from it.
```

Reply:
Maybe refer to the section where each OS's directory is listed as this directory is only for linux